pom.xml

```diff
@@ -50,7 +50,7 @@
     <properties>
         <sirocco.version>1.0</sirocco.version>
 
-        <powsybl-ws-dependencies.version>2.12.0</powsybl-ws-dependencies.version>
+        <powsybl-ws-dependencies.version>2.13.0-SNAPSHOT</powsybl-ws-dependencies.version>
```

Comment:
No use of snapshot version here
Instead, wait for new network-store client release containing the changes made in this PR :  https://github.com/powsybl/powsybl-network-store/pull/424

And then override the network-store client version with this new release in this pom.xml, the same way it was done in the pom.xml in this PR : https://github.com/powsybl/powsybl-network-store-server/pull/56

Reply:
yes that was to local test